### ISSUE TITLE
Include passenger_max_requests in the Standalone template

### ISSUE THREAD

**Issue by scottjacobsen** [phusion/passenger#1625]
The [standalone template](https://github.com/phusion/passenger/blob/7e547f4002d3e2c0df0b0acb6c6b5ad04103e38f/resources/templates/standalone/config.erb) doesn't include the `passenger_max_requests` option, so I need to copy the template and modify it. 

That is a somewhat dangerous operation according to the documentation, and requires manual intervention if the template changes.

`passenger_max_requests` seems like a useful option to include in the default template. Please add it if you can. Thanks.

**Comment by scottjacobsen** [comment]
If this could become a configuration option for passenger standalone I could nuke my my custom nginx.conf.erb and I would be super happy :)

**Comment by FooBarWidget** [comment]
Starting from 5.0.22, the Nginx configuration template has been slimmed down so that it is much easier to customize it without breaking compatibility with future versions:

https://github.com/phusion/passenger/blob/9346256f895c7196833f01dba2475ef53839d0b9/resources/templates/standalone/config.erb

**Comment by thomasdavis** [comment]
```bash
web: ./passenger-status-service-agent & bundle exec passenger start -p $PORT --startup-file node/node --app-type node --max-pool-size 2 --passenger-max-requests 100
```

I would love to be able to do the above. Does anyone know if this is possible or when it will be?

**Comment by OnixGH** [comment]
If it's not listed in `passenger start --help` then it's not possible.

We haven't planned anything but feel free to have a look at adding it through `config_options_list.rb`, `builtin_engine.rb` (add_param) and `OptionParser.h`.

**Comment by nightsurge** [comment]
With the new changes to the config.erb, my app is now saying that "max_requests" is an invalid option. How do I regain this functionality?

`*** WARNING: Passengerfile.json: configuration key 'max_requests' is not supported`

I used to use an option like this:

`<%= nginx_option :passenger_max_requests, :max_requests %>`
but I tried re-adding this to the new template copy and it keeps throwing the error above.

**Comment by FooBarWidget** [comment]
@nightsurge Are you on open source or enterprise?

**Comment by nightsurge** [comment]
@FooBarWidget currently open source.

**Comment by FooBarWidget** [comment]
This is definitely our bug in our options registry. It looks like I forgot to add this one during the refactoring.

**Comment by nightsurge** [comment]
@FooBarWidget It does appear that the feature still works, even though the logger reports that the option is not valid. I've been watching my server all day and memory has not grown out of control and my processes appear to be spawning/closing down at the correct times. So it may all just be a bug in whatever generates those warning dialogs.

Thanks!

**Comment by thomasdavis** [comment]
@nightsurge are you saying `passenger --max-requests 1000` throws a warning but actually works? Or are you using the nginx conf.

**Comment by nightsurge** [comment]
@thomasdavis sorry for the confusion, referring to my specific bug/issue I tagged into this thread a few posts back. Still using nginx.config.erb.

**Comment by thomasdavis** [comment]
If anybody is watching this issue, I am trying to run Node.js on Heroku and need this option to bandaid memory leaks. I've scoured the internet for a `--nginx-config-template nginx.conf.erb` that works on Heroku with Node.js but have failed.

It would be really easy to do if this command was added to the CLI so I wouldn't have to setup a nginx.conf.

If no one has any suggestions I will give it my best shot to submit a pull request.

**Comment by OnixGH** [comment]
@thomasdavis did you see the suggestions in my previous post?

**Comment by thomasdavis** [comment]
@OnixGH Yep, doesn't look too complicated, might get to it on the weekend.
